### ISSUE TITLE
wrong uncl1153 namespace

### ISSUE THREAD

**Issue by VladimirAlexiev** [w3c-ccg/traceability-vocab#268]
Several schemas (eg https://github.com/w3c-ccg/traceability-vocab/blob/main/docs/openapi/components/schemas/common/BillOfLading.yml)
use wrong URLs for uncl1153 props, eg
`https://service.unece.org/trade/uncefact/vocabulary/uncl1153/#Consignment_identifier_carrier_assigned`.

The rights URL use `#` not `/#`, eg
`https://service.unece.org/trade/uncefact/vocabulary/uncl1153#Consignment_identifier_carrier_assigned`.

As you can see here: https://service.unece.org/trade/uncefact/vocabulary/uncl1153.jsonld.

BTW, it's much better to define prefix `uncl1153:` in the JSONLD context,
and then use CURIEs (eg  `uncl1153:Consignment_identifier_carrier_assigned`)
in `$linkedData.@id`. But is that allowed in the JSON Schema that you use?

**Comment by OR13** [comment]
https://www.w3.org/TR/cooluris/

Let's make a concrete proposal, or close this issue.

**Comment by brownoxford** [comment]
Discussed, and the working group has decided not to use CURIEs, closing.

**Comment by VladimirAlexiev** [comment]
@brownoxford Maybe you meant to close #285? But please provide some justification or point to some minutes!

This issue is about some of the schemas using wrong URLs. So reopen it.

**Comment by brownoxford** [comment]
@VladimirAlexiev please see new issue #536

**Comment by VladimirAlexiev** [comment]
@brownoxford This issue is still not fixed, please reopen it.

**Comment by brownoxford** [comment]
@VladimirAlexiev, are you saying that #536 does not accurately reflect what needs to happen here, or that you disagree with waiting for the vocab v1 bump later this year? @nissimsan is point on this issue currently, and his suggestion is: "I advise that we don't start this work until the UN vocab is published"

**Comment by nissimsan** [comment]
... And to qualify that, @brownoxford and @TallTed , my aim is to stick with a single URL for defining terms. 

`https://service.unece.org/trade/uncefact/vocabulary/uncl1153/#`
and 
`https://service.unece.org/trade/uncefact/vocabulary/uncl1153#`
are not the same. They produce different RDF nodes, and changing them breaks proofs. 

Despite the uncoolness factor, I consistently stick with the URL which I get redirected to as a general practice or convention. I don't consider this a problem for us (trace-vocab) to be solving - it should be solved by the vocab we're referencing (in this case UN cefact). And it is.

**Comment by TallTed** [comment]
> I consistently stick with the URL which I get redirected to as a general practice or convention

That will break Linked Data, consistently and absolutely, and I will block it by any means necessary.

**Comment by nissimsan** [comment]
@TallTed, your argument on https://github.com/uncefact/spec-jsonld/issues/25#issuecomment-1227414233 has changed my mind on this. Thank you!

**Comment by VladimirAlexiev** [comment]
@brownoxford I don't know what are the future plans, but this bug should stay open until it is fixes. 
I just checked, the bug is still present.
